hack/get-admin-kubeconfig.sh

```diff
@@ -1,8 +1,21 @@
 #!/bin/bash
+set -u
 
-if [[ "$#" -ne 1 ]]; then
-    echo "usage: $0 resourceid" >&2
+RESOURCEID=$1
+CONTEXT_NAME=$2
+
+if [[ "$#" -ne 2 ]]; then
+    echo "usage: $0 resourceid context_name" >&2
     exit 1
 fi
 
-go run ./hack/db "$1" | jq -r .openShiftCluster.properties.adminKubeconfig | base64 -d | sed -e 's|https://api-int\.|https://api\.|'
+if [[ "$CONTEXT_NAME" == "admin" ]]; then
+  KEY="adminKubeconfig"
+elif [[ "$CONTEXT_NAME" == "aro-service" ]]; then
+  KEY="aroServiceKubeconfig"
+else
+  echo "usage: context name must be one of admin or aro-service"
+  exit 1
+fi
```

Comment:
why do we need this change, i.e. under what circumstances do we want to get the aro-service kubeconfig?